gym_ergojr/envs/ergo_reacher_augmented.py

```diff
@@ -86,19 +93,23 @@ def step(self, action):
 
         obs_diff = self.augment(obs, action, new_obs)
 
-        corrected_obs = (new_obs[:, :8] +
-                         obs_diff[:, [1, 2, 4, 5, 7, 8, 10, 11]]).numpy()
-
+        corrected_obs = new_obs[:, :8] + obs_diff[:, [1, 2, 4, 5, 7, 8, 10, 11]]
+        new_obs[:, :8] = corrected_obs
+        if self.is_cuda:
```

Comment:
You can remove the conditional here: calling `.cpu()` on something that is already on the CPU doesn't cause an issue. In fact, you can call `tensor.cpu().cpu().cpu()` without any problems.

Reply:
oh! I didn't know that! Thanks